app/fee_distribution.go

```diff
@@ -23,8 +23,9 @@ func NewValAddrCache(stakeKeeper stake.Keeper) *ValAddrCache {
 }
 
 type ValAddrCache struct {
-	cache       map[string]sdk.AccAddress
-	stakeKeeper stake.Keeper
+	cache                 map[string]sdk.AccAddress
+	distributionAddrCache map[string]sdk.AccAddress
+	stakeKeeper           stake.Keeper
 }
 
 func (vac *ValAddrCache) ClearCache() {
```

Comment:
Missed to clean `distributionAddrCache`

Reply:
can we force clean cache when it is breath block?

Reply:
> Missed to clean `distributionAddrCache`

added

> can we force clean cache when it is breath block?

sure. added.